### PR TITLE
fix: Automation AccessibilityId: likeButton is missing

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Set MessageToolboxView's isAccessibilityElement property to false to allow like button can be accessed.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift [hunk 1]
```diff
@@ -66,7 +65,7 @@ import TTTAttributedLabel
     override init(frame: CGRect) {
         
         super.init(frame: frame)
-        self.isAccessibilityElement = true
+        isAccessibilityElement = false
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift]: @alexaubry @billypchan would this be still OK for the actual VoiceOver users?

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift]: The MessageToolboxView's accessibilityValue will be set to its subview's statusLabel's value in several places:
https://github.com/wireapp/wire-ios/blob/de6cc3a42ff36149c38e21384e021796be780430/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift#L419

After this fix the status label is accessible and I updated its accessibility value in the commit below.

Comment (alexaubry) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift]: @mikeger I will test on device

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift]: @alexaubry @billypchan  any update on this?

**Comment by alexaubry** [comment]
@mikeger @typfel This sounds good on device. We will need to revisit the accessibility behavior of the message toolbox later anyway, as there is no way to show it in Voice Over at the moment.

**Pull request merged**